cibuildwheel/windows.py

```diff
@@ -100,40 +100,44 @@ def shell(args, env=None, cwd=None):
 
     python_configurations = get_python_configurations(build_selector)
     for config in python_configurations:
+        # install Python
         config_python_path = get_python_path(config)
         simple_shell([nuget, "install"] + get_nuget_args(config))
-        if not os.path.exists(os.path.join(config_python_path, 'Scripts', 'pip.exe')):
-            simple_shell([os.path.join(config_python_path, 'python.exe'), get_pip_script ])
-
-        # check python & pip exist for this configuration
         assert os.path.exists(os.path.join(config_python_path, 'python.exe'))
-        assert os.path.exists(os.path.join(config_python_path, 'Scripts', 'pip.exe'))
-
-        # setup dirs
-        if os.path.exists(built_wheel_dir):
-            shutil.rmtree(built_wheel_dir)
-        os.makedirs(built_wheel_dir)
 
+        # set up PATH and environment variables for run_with_env
         env = os.environ.copy()
-        # set up environment variables for run_with_env
         env['PYTHON_VERSION'] = config.version
         env['PYTHON_ARCH'] = config.arch
         env['PATH'] = os.pathsep.join([
             config_python_path,
             os.path.join(config_python_path, 'Scripts'),
             env['PATH']
         ])
+        # update env with results from CIBW_ENVIRONMENT
         env = environment.as_dictionary(prev_environment=env)
 
         # for the logs - check we're running the right version of python
+        shell(['where', 'python'], env=env)
         shell(['python', '--version'], env=env)
         shell(['python', '-c', '"import struct; print(struct.calcsize(\'P\') * 8)\"'], env=env)
```

Comment:
These shall probably use the `simple_shell`

Reply:
Thanks; that's what I thought, but they weren't before, so I wasn't sure :)